pkg/antctl/antctl.go

```diff
@@ -129,12 +129,17 @@ var CommandList = &commandList{
 					params: []flagInfo{
 						{
 							name:  "name",
-							usage: "Get NetworkPolicy by name. If present, Namespace must be provided.",
+							usage: "Get NetworkPolicy by name.",
 							arg:   true,
 						},
+						{
+							name:      "source",
+							usage:     "Get Networkpolicies with specific source. Source means the name of its source network policy.",
+							shorthand: "S",
+						},
 						{
 							name:      "namespace",
-							usage:     "Get Networkpolicies from specific Namespace",
+							usage:     "Get Networkpolicies from specific Namespace.",
```

Comment:
Since the internal NP is changed to cluster scope, maybe something like `sourceNamespace` makes more sense here?

Reply:
Thanks for the suggestion. I considered this too, however the namespace is also used to filter NetworkPolicies applied to the Pod along with the "pod" parameter so I didn't change it to "source" specific.

Reply:
I see thanks for the explanation.